src/components/OneTimePassword/OneTimePassword.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useRef, KeyboardEvent } from 'react'
+import classNames from 'classnames'
+import { Input } from '../Input'
+import { OtpCustomProps } from './types'
+import { ErrorMessage, Label } from '../../helperComponents'
+import { Text } from '../Text'
+import IconCheckmarkCircleFilled from '../SVGIcons/IconCheckmarkCircleFilled'
+
+export const OneTimePassword = React.forwardRef<HTMLInputElement, OtpCustomProps>(
+  ({
+    className,
+    size = 'large',
+    error,
+    hasError,
+    label,
+    name,
+    disabled,
+    required,
+    readonly,
```

Comment:
Gar jan is state of disabled and readonly same? can you use one of them for all cases?